### ISSUE TITLE
Missing babel-preset-vue-app

### ISSUE THREAD

**Issue by bmaland** [egoist/create-vue-app#21]
Booting a new app created using `create-vue-app` fails with the following error:

```
missing babel preset: following babel preset is not found in ~/tmp/myvueapp:

- babel-preset-vue-app
```

I believe it is caused by this commit: https://github.com/egoist/poi/commit/813e6b3a7b4048a2606f8e6c85469fa768e33e85

**Comment by egoist** [comment]
ah yeah we replaced it with babel-preset-poi, update it https://github.com/vue-land/create-vue-app/blob/05e04fcf6014e2f12dab7384b0d0f7e4be080ea9/packages/template-vue/template/package.json#L16-L20 here would fix this, can you do a PR 😄

**Comment by egoist** [comment]
in fact we can just remove these lines

**Comment by egoist** [comment]
Fixed in `v1.1.11`
